.github/workflows/ft_tests.yml

```diff
@@ -0,0 +1,40 @@
+name: Free Threading Tests
+on: [push, pull_request]
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install uv and set the python version
+        uses: astral-sh/setup-uv@v5
+        with:
+          version: "0.7.2"
+          python-version: 3.13t
+          enable-cache: false
+      - name: Set up Python
+        run: uv python install
+      - name: Install Dependencies
+        run: |
+          set -x
+          set -e
+          uv pip install packaging setuptools pyflakes pytest-cov pytest-flakes pytest-timeout pytest-durations
+          # Test dep, consider skipping
+          uv pip install sympy hypothesis
+          uv pip install --pre --upgrade -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple cython numpy
+          uv pip install pytest-run-parallel
+      - name: Disable Coverage # see gh-198
+        run: |
+          sed -i '/plugins = Cython.Coverage/d' .coveragerc
+          sed -i 's/addopts/\#addopts/' pytest.ini
+      - name: In-place build
+        run: |
+          uv pip install . --no-build-isolation -v
+          uv run --no-project python setup.py clean
+          uv run --no-project  python setup.py build_ext --inplace
```

Comment:
This seems wrong and doing a simple editable install in a single invocation is the right way to do this. However, I see that it's copied from `tests.yml` where the same thing happens, so there may be a reason. I think it has to do with Cython coverage differing between Python <3.12 and >=3.12, but that's not relevant here.

Okay to not investigate - I know you're short on time.